introduction_to_repository.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to the Repository
```

Comment:
It will be great to have this introduction, but let's introduce it in a separate PR because it mostly doesn't relate to rid_qualifier.

Reply:
Yes agreed, this was a error in my part, apologies. I have sent a new PR for this.